src/AuthenticationRequest.ts

```diff
@@ -133,6 +135,14 @@ function createURIFromJWT(
   jwt: string
 ): SIOP.AuthenticationRequestURI {
   const schema = 'openid://';
+  const pejs: PEJS = new PEJS();
```

Comment:
Suggest to move this to a function and do the actual validation in the createJWT method instead of in the createURIFromJWT function, as that only is creating the URI as its responsibility. In createJWT the payload is created and it is the method that always will be called eventually.